modules/working-with-accelerator-profiles.adoc

```diff
@@ -0,0 +1,53 @@
+:_module-type: CONCEPT
+
+[id='working-with-accelerator-profiles_{context}']
+= Working with accelerator profiles
+
+[role='_abstract']
+To configure accelerators for your data scientists to use in {productname-short}, you must create an associated accelerator profile. An accelerator profile is a custom resource definition (CRD) on OpenShift that has an AcceleratorProfile resource, and defines the specification of the accelerator.  
+
+For accelerators that are new to your deployment, you must manually configure an accelerator profile for the accelerator. If you use NVIDIA GPUs, an accelerator profile generates automatically after you upgrade to the latest version of {productname-short}, and resides in the `Instances` section of the `acceleratorprofiles.dashboard.opendatahub.io` custom resource definition (CRD). 
+
```

Comment:
Here I miss everything related with upgrading the operator:
If the user had already GPUs working in the  {productname-short} older version: The AcceleratorPRofile CRD instances will be created automatically.
If the user add GPUs after upgrading or after an operator fresh install of the new version: The acceleratorProfile CRD instances should be created manually

Reply:
@chtyler This one is still unfinished?

Reply:
Now fixed.